website/content/docs/integrate/partnerships.mdx

```diff
@@ -37,7 +37,7 @@ By leveraging Consul's RESTful HTTP API system, prospective partners are able to
 
 -> **Network Infrastructure Automation (NIA)***: These integrations leverage Consul's service catalog to seamlessly integrate with Consul-Terraform-Sync (CTS) to automate changes in network infrastructure via a publisher-subscriber method. More details can be found [here](/docs/integrate/nia-integration).
 
-**HCP Consul**: HCP Consul is secure by default and offers an enterprise-level service level agreement (SLA) to deploy an organization's most important applications. Sign up for HCP Consul is free and available [here](https://cloud.hashicorp.com/products/consul).
+**HCP Consul**: HCP Consul is secure by default and offers an out-of-the-box service mesh solution to streamline operations without the hassle of managing Consul servers. Sign up for HCP Consul is free and available [here](https://cloud.hashicorp.com/products/consul).
```

Comment:
```suggestion
**HCP Consul**: HCP Consul is secure by default and offers an out-of-the-box service mesh solution to streamline operations without the hassle of managing Consul servers. [Sign up for a free HCP Consul account](https://cloud.hashicorp.com/products/consul).
```
It's good practice to link keywords and phrases, rather than "here", "this link", etc.